src/fit-utils.jl

```diff
@@ -24,7 +24,7 @@ function get_edges(X::AbstractMatrix{T}; fnames, nbins, rng=Random.MersenneTwist
 end
 
 function get_edges(df; fnames, nbins, rng=Random.MersenneTwister())
-    _nobs = length(Tables.getcolumn(df, 1))
+    _nobs = Tables.DataAPI.nrow(df)
```

Comment:
Yeah, tables are not currently required to implement `DataAPI.nrow`, or something like it. This is a long standing issue. To be honest, I'd be happy to restrict to tables that that do implement `DataAPI.nrow` and push for implementations on a case-by-case basis. 

Of course my point is moot, if you only apply `get_edges` to column tables (output of `Tables.columntable`).

Reply:
Thanks for pointing this out. Given that I enforce input columntable, then my understanding is this can be let as it is.